bin/inx-chronicle/src/main.rs

```diff
@@ -172,7 +182,11 @@ async fn main() {
 }
 
 async fn startup(scope: &mut RuntimeScope) -> Result<(), Box<dyn Error + Send + Sync>> {
-    let launcher_addr = scope.spawn_actor(Launcher).await;
+    let launcher = Launcher {
+        inx_connection_retry_interval: std::time::Duration::from_secs(5),
```

Comment:
Should this be part of the Config?

Reply:
Yes, but I'm leaving that for @Alex6323 :trollface:

Reply:
thank you :heart: